docs/source/guides/feature_selection.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feature Selection\n",
+    "\n",
+    "Featuretools provides users with the ability to remove features that are unlikely to be useful in building an effective machine learning model. Reducing the number of features in the feature matrix can both produce better results in the model as well as reduce the computational cost involved in prediction.\n",
+    "\n",
+    "Featuretools enables users to perform feature selection on the results of Deep Feature Synthesis with three functions:\n",
+    "\n",
+    "- `ft.selection.remove_highly_null_features`\n",
+    "- `ft.selection.remove_single_value_features`\n",
+    "- `ft.selection.remove_highly_correlated_features`\n",
+    "\n",
+    "Each of the functions takes in a calculated feature matrix and can optionally take in the list of Feature definitions as well. If just a feature matrix is used as input, then just a feature matrix will be returned, but if both the matrix and the Feature definitions are used, then the results of feature selection will, similarly, be the matrix and the Feature definitions.\n",
+    "\n",
+    "We will describe each of these functions in depth, but first we must create an entity set with which we can run `ft.dfs`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import featuretools as ft\n",
+    "\n",
+    "from featuretools.selection import (\n",
+    "    remove_highly_correlated_features,\n",
+    "    remove_highly_null_features,\n",
+    "    remove_single_value_features,\n",
+    ")\n",
+    "\n",
+    "from featuretools.primitives import NaturalLanguage\n",
+    "\n",
+    "\n",
+    "df1 = pd.DataFrame({'id': [0, 1, 2, 3], \n",
+    "                    'categories':['a','a','b','b'], \n",
+    "                    'bools':[True, True, False, False],\n",
+    "                    'half_nulls': [None, None, 88, 100],\n",
+    "                    'quarter_nulls': [None, 1,1, 1],\n",
+    "                    'all_nulls': pd.Series([None, None, None,None], dtype='float'),\n",
+    "                    \"diff_ints\": [34, 11, 29, 91],\n",
+    "                   \n",
+    "                   })\n",
+    "df2 = pd.DataFrame({\n",
+    "    'id': [0, 1, 2, 3],\n",
+    "    \"first_id\": [0, 1, 1, 3],\n",
+    "    \"words\": [\"test\", \"this is a short sentence\", \"foo bar\", \"baz\"],\n",
+    "    \"corr_words\": [4, 24, 7, 3],\n",
+    "    'corr_1': [99, 100, 77, 33],\n",
+    "    'corr_2': [99, 100, 77, 33],\n",
+    "})\n",
+    "\n",
+    "entities = {\n",
+    "        \"first\": (df1, 'id'),\n",
+    "        \"second\": (df2, 'id',  None, {'words': NaturalLanguage}),\n",
+    "    }\n",
+    "\n",
+    "es = ft.EntitySet(\"data\", entities, )\n",
```

Comment:
i wonder if we should just use one of the demo datasets? perhaps we need to add a demo dataset with nulls. 

i think that it's better for a examples to use standardize datasets. mostly for the user to have consistency as they read the docs, but also to avoid potential mistakes like the example her missing a relationship between the entities